query/sql/sql_instance_postgresql_log_statement_database_flag_set_appropriately.sql

```diff
@@ -0,0 +1,22 @@
+select 
```

Comment:
Change query name to `sql_instance_postgresql_log_statement_database_flag_ddl.sql`